core-services/release-controller/_releases/release-ocp-4.7-ci.json

```diff
@@ -54,6 +54,11 @@
       "upgradeFrom": "PreviousMinor",
       "optional":false,
       "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade"}
+    },
+    "upgrade-minor-ovn":{
+      "upgrade":true,
```

Comment:
Needs `"upgradeFrom: "PreviousMinor",`

Reply:
done

Reply:
It was added for the periodic jobs. This one (in verify) was missed

Reply:
yep, thanks. fixed it.